content/jobs/README.md

```diff
@@ -0,0 +1,46 @@
+# Job Post Instructions
+
+---
+
+Hey there! If you're a local-ish tech company looking to hire talent we at NewHaven.io would love to help out by putting some information about your organization and open role(s) on our website.
+
+Just clone this repository, add your information to [this page](https://github.com/newhavenio/newhavenio/tree/master/content/jobs/jobs.yaml), and submit a pull request.
+
+```markdown
+- company: "NewHaven.IO"
+  jobURL: "https://newhaven.io"
+  jobTitle: "Community Moderator"
+  location: "New Haven, CT // Remote"
+  tech: "Slack, MeetUp, GatsbyJS"
+  salary: "Volunteer"
+  description: "We're looking for help with some community moderation. As vaccines keep rolling out we're looking at ways we can begin to breathe life back into the group. We want to wade back into holding some events and we're going to need help keeping an eye our Slack team and Meetup page. Over time we can fold whoever takes this role into our organizing and leadership teams. If you've gotten some value from IO in the past we'd love your help in carrying it forward."
+```
```

Comment:
I feel like we need an expires at field to avoid the toil of cleanup

Reply:
Personally I want to see if this takes off before I put in the engineering time for an expiration field.

Getting people to use the right date format is going to be a pain in the ass.

Reply:
Well, sure! I'm always down for a wait and see approach to scope creep. That said we literally review each PR and can request date changes

Reply:
> That said we literally review each PR and can request date changes

Fair enough. I'll add an expiration date field and add in the README that we suggest like a 90 day window and will go in on a regular cadence and remove things. But that they're open to resubmit?

Reply:
Added and pushed.